geoutils/georaster/raster.py

```diff
@@ -2043,63 +2068,93 @@ def format_value(value: Any) -> Any:
                 value = None
             return value
 
-        # Need to implement latlon option later
-        if latlon:
-            from geoutils import projtools
+        # Initiate output lists
+        list_values = []
+        if return_window:
+            list_windows = []
 
-            x, y = projtools.reproject_from_latlon((y, x), self.crs)
+        # Loop over all coordinates passed
+        for k in range(len(x)):  # type: ignore
 
-        # Convert coordinates to pixel space
-        row, col = rio.transform.rowcol(self.transform, x, y, op=round)
+            value: float | dict[int, float] | tuple[float | dict[int, float] | tuple[list[float], np.ndarray] | Any]
 
-        # Decide what pixel coordinates to read:
-        if window is not None:
-            half_win = (window - 1) / 2
-            # Subtract start coordinates back to top left of window
-            col = col - half_win
-            row = row - half_win
-            # Offset to read to == window
-            width = window
-            height = window
-        else:
-            # Start reading at col,row and read 1px each way
-            width = 1
-            height = 1
+            if latlon:
+                from geoutils import projtools
 
-        # Make sure coordinates are int
-        col = int(col)
-        row = int(row)
+                x0, y0 = projtools.reproject_from_latlon((y[k], x[k]), self.crs)  # type: ignore
+            else:
+                x0 = x[k]  # type: ignore
+                y0 = y[k]  # type: ignore
+
+            # Convert coordinates to pixel space
+            row, col = rio.transform.rowcol(self.transform, x0, y0, op=round)
```

Comment:
Same, I think this can work on a list directly, rather than looping...

Reply:
Done